### PR TITLE
Fb page format

### LINKED ISSUE #17: Format des pages
En tant qu’utilisateur, j’aimerai pouvoir créer des pages contenant des métadonnées et du contenu.

### PR DESCRIPTION
**Pull request by jlheig**
Added a section discussing our choices and template for the format of article pages inside the README.md file.
This should close #17

### CODE REVIEW

**Review by mathias-maillard — DISMISSED** [review]
This looks good :)

**Comment by pozniako16** [comment]
Be careful, the discussion is in the portfolio already

File: README.md [hunk 1]
```diff
@@ -2,3 +2,30 @@ Membre: Mathias Maillard, Jean-Luc Blanc, Lev Pozniakoff
 
 # GEN: Labo1
```
Comment (pozniako16) [inline README.md:3]: We can remove that :)

File: README.md [hunk 2]
```diff
@@ -2,3 +2,30 @@ Membre: Mathias Maillard, Jean-Luc Blanc, Lev Pozniakoff
...
 
+
+
+
+
+## Format des pages articles
```
Comment (pozniako16) [inline README.md:9]: too many spaces

**Review by pozniako16 — CHANGES_REQUESTED** [review]
<3

**Review by pozniako16 — APPROVED** [review]

**Pull request merged**